news/_posts/2022-10-03-hacktoberfest-2022-started.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Hacktoberfest and PrestaShop: how you can participate in this 2022 edition"
+subtitle: "October is here: participate in Hacktoberfest, a celebration of open source software!"
+date:  2022-10-03
+authors: [ PrestaShop ]
+image: /assets/images/2022/09/hacktoberfest2022.png
+icon: icon-github-alt2
+tags: [community, contribution, github, contribute]
+---
+
+October is now here and Hacktoberfest, a month-long celebration of open source software, has started. We have recently written an article about the event, how you can get involved and all the different rules. You can know more [in our Hacktoberfest 2022 article](/news/hacktoberfest-2022/).
+
+In a few words, participation is very simple: people are encouraged to create at least 4 pull requests for open source projects on GitHub.
+
+![Hacktoberfest 2022](/assets/images/2022/09/hacktoberfest2022.png)
+## How to get started
+
+If you are wondering how to get started, we are listing in this blog post issues that could be a good starting point for new contributors. Also added to the list are issues needing to be fixed, so you can easily contribute.
+
+List of issues for new contributors:
+
+TODO
+
+List of issues needing to be fixed:
```

Comment:
hey @PrestaShop/product-team 

Are there any issues that you would like to highlight in this place?